<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BhagyasriSagar" ignored="0" name="firefoxTest" tests="1" failures="1" timestamp="18 Nov 2020 00:08:16 GMT" time="12.245" errors="0">
  <testcase name="loginTest" time="5.423" classname="com.nopcommerce.testCases.TC_LoginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.nopcommerce.testCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:38)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- firefoxTest -->
